llvm/test/CodeGen/Generic/builtin-expect-with-probability.ll

```diff
@@ -1,4 +1,5 @@
 ; RUN: llc < %s
+; RUN: llc -global-isel < %s
 
```

Comment:
The existing test is woefully inadequate. Should fix this in a pre-commit. This needs to have checks, and should not be a "Generic" test. Those cannot exist

Reply:
For generic changes like this, is it sufficient to test against one backend or does each one need its own coverage?

Reply:
I have moved testing for GISel to the AArch64 IRTranslator no-op intrinsics suite, where similar tests lie, and removed the update to the generic one. I'll update the SelectionDAG tests to be specific and remove the generic one altogether separately.